domain/optatask/src/main/java/org/optatask/domain/Task.java

```diff
@@ -138,6 +139,7 @@ public Integer getStartTime() {
 
     public void setStartTime(Integer startTime) {
         this.startTime = startTime;
+        this.endTime = startTime == null ? null : startTime + getDuration();
```

Comment:
I realized that best solution is obtained directly from KIE Server so we're bound to work with the planning domain (unless we add an extra layer, maybe by creating a KIE Server extension?). So however this might be difficult to avoid let's at least add a `FIXME` note.

Reply:
Using non-simple setters is dangerous, especially with frameworks such as jackson and JAXB that tend to go through the getters+setters instead of the fields. I shun non-simple setters.

Instead, I tend to do the following:
- Either have a getter+setter for start and end, so then I have only a getter for duration (which returns end - start)
- Xor have a getter+setter for start and duration, and only have a getter for end (which returns start+ duration)

Reply:
@ge0ffrey @yurloc I made endTime another customShadowVariable so it's sent over REST API, wdyt?